charts/stable/minecraft-java/questions.yaml

```diff
@@ -293,237 +331,244 @@ questions:
           schema:
             type: boolean
             default: true
         - variable: VIEW_DISTANCE
-          label: "Server View Distance"
+          label: Server View Distance
           schema:
             type: int
             required: true
             default: 10
         - variable: SEED
-          label: "Level Seed"
+          label: Level Seed
           schema:
             type: string
             default: ""
         - variable: MODE
-          label: "Default Gamemode"
+          label: Default Gamemode
           schema:
             type: string
             required: true
-            default: "survival"
+            default: survival
             enum:
-              - value: "survival"
-                description: "Survival"
-              - value: "creative"
-                description: "Creative"
-              - value: "adventure"
-                description: "Adventure"
-              - value: "spectator"
-                description: "Spectator"
+              - value: survival
+                description: Survival
+              - value: creative
+                description: Creative
+              - value: adventure
+                description: Adventure
+              - value: spectator
+                description: Spectator
         - variable: MOTD
-          label: "Server Message of the Day"
+          label: Server Message of the Day
           schema:
             type: string
-            default: "Welcome to Minecraft on TrueNAS Scale!"
+            default: Welcome to Minecraft on TrueNAS Scale!
         - variable: PVP
-          label: "Enable/Disable PVP"
+          label: Enable/Disable PVP
           schema:
             type: boolean
             default: false
         - variable: LEVEL_TYPE
-          label: "Level Type"
+          label: Level Type
           schema:
             type: string
             required: true
-            default: "DEFAULT"
+            default: DEFAULT
         - variable: GENERATOR_SETTINGS
-          label: "Generator Settings"
+          label: Generator Settings
           schema:
             type: string
             default: ""
         - variable: LEVEL
-          label: "World Save Name"
+          label: World Save Name
           schema:
             type: string
             required: true
-            default: "world"
+            default: world
         - variable: WORLD
-          label: "URL of world download"
+          label: URL of world download
           schema:
             type: string
             default: ""
         - variable: FORCE_REDOWNLOAD
-          label: "Force redownload server files"
+          label: Force redownload server files
+          description: Force re-download of the server file for VANILLA, FORGE, BUKKIT, SPIGOT, PAPER, CURSEFORGE, SPONGEVANILLA server types.
           schema:
-            type: string
-            default: ""
+            type: boolean
+            default: false
```

Comment:
I'm not sure if upgrade will be happy on this. Need to make sure first

Reply:
looks like its cleared.

Reply:
No I mean, If someone has installed the app where this value was previously a `string` (either `false` or `true`).
While upgrading it will probably show an error complaining that expected string but found boolean, or something like that.

that will only be able to verify live

Reply:
how can i test it? im currently running a testing mc-java chart

Reply:
im running two instances with a empty string and a `false` as values

Reply:
honestly, we can just leave it as string type and eventually move it as a boolean when a major release happens.

Reply:
Just make it strong and put enum 
With true and false

Reply:
So a dict?